docs/platforms/javascript/common/configuration/filtering.mdx

```diff
@@ -213,13 +213,15 @@ If the transaction currently being processed has a parent transaction (from an u
 
 Learn more about <PlatformLink to="/configuration/sampling/">configuring the sample rate</PlatformLink>.
 
-### Using <PlatformIdentifier name="before-send-transaction" />
+### Using `beforeSendTransaction`
```

Comment:
I changed this to the plain string because 1. I don't think `<PlatformIdentifier>` does anything here since it's a JS-specific docs page and 2. I can't link to this heading properly because of https://github.com/getsentry/sentry-docs/issues/10613. 

Happy to wait with merging this PR until the issue is fixed but I still don't see why we need PlatformIdentifier at all here.